ROADMAP.md

```diff
@@ -77,9 +77,9 @@ https://github.com/libp2p/rust-libp2p/issues/2883.
 
 ### WebRTC support (browser-to-server)
 
-| Category     | Status      | Target Completion | Tracking                                 | Dependencies                                   | Dependents |
-|--------------|-------------|-------------------|------------------------------------------|------------------------------------------------|------------|
-| Connectivity | In progress | Q4/2022           | https://github.com/libp2p/specs/pull/412 | https://github.com/libp2p/test-plans/issues/53 |    [WebRTC (browser-to-browser)](#webrtc-support-browser-to-browser)        |
+| Category     | Status | Target Completion | Tracking                                 | Dependencies                                   | Dependents                                                        |
+|--------------|--------|-------------------|------------------------------------------|------------------------------------------------|-------------------------------------------------------------------|
+| Connectivity | Done   | Q4/2022           | https://github.com/libp2p/specs/pull/412 | https://github.com/libp2p/test-plans/issues/53 | [WebRTC (browser-to-browser)](#webrtc-support-browser-to-browser) |
```

Comment:
```suggestion
| Connectivity | Done   | Q4/2022           | https://github.com/libp2p/specs/pull/412 | https://github.com/libp2p/test-plans/pull/100 | [WebRTC (browser-to-browser)](#webrtc-support-browser-to-browser) |
```
Perhaps it's better to link directly to the PR adding interop tests?

(note: it probably makes sense to close https://github.com/libp2p/test-plans/issues/53 I'll do this)